### DISCUSSION TITLE
Replacement for Jyers firmware?

### DISCUSSION THREAD

**Discussion by TheStork239** [Jyers/Marlin#2046] [General Discussion]
Hi all,
As it sadly seems that his initiative has run its course, with no new firmware for nearly a year, what are you now using instead to keep the firmware current?
I loved the fact that with Jyers there were pre-compiled firmwares ready to go (for me a 4.2.2 board with BLTouch and 5x5 mesh), as tinkering with the config files and compiling my own was not what I had in mind. 
It seems that Marlin firmware has continued to develop, but creating compiled builds with BLTouch would for me require major time investment and potential trial and error to get the firmware updated.
Have there been any similar projects come on like this one that you have moved onto?

Finally, just wanted to say thank you @Jyers for the work you put into this initiative.  Really appreciate the work you did, and understand that it may have been time to move onto other projects.

**Comment by HAPSagan** [comment]
Hi I am somewhat in the same position as you. I can inform you that the
project has been taken over by another and lives on. It's been moved to
discord but is only in French now, which makes it pretty much impossible
for me to understand.

Den lör 1 okt. 2022 kl 13:58 skrev TheStork239 ***@***.***>:

> Hi all,
> As it sadly seems that his initiative has run its course, with no new
> firmware for nearly a year, what are you now using instead to keep the
> firmware current?
> I loved the fact that with Jyers there were pre-compiled firmwares ready
> to go (for me a 4.2.2 board with BLTouch and 5x5 mesh), as tinkering with
> the config files and compiling my own was not what I had in mind.
> It seems that Marlin firmware has continued to develop, but creating
> compiled builds with BLTouch would for me require major time investment and
> potential trial and error to get the firmware updated.
> Have there been any similar projects come on like this one that you have
> moved onto?
>
> Finally, just wanted to say thank you @Jyers <https://github.com/Jyers>
> for the work you put into this initiative. Really appreciate the work you
> did, and understand that it may have been time to move onto other projects.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Jyers/Marlin/discussions/2046>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATXWKRDP32E22QEJXY2GGQLWBARP7ANCNFSM6AAAAAAQ2NPNII>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Med vänliga hälsningar!

Nina och Peter Lindh

**Reply by darco415** [reply]
thank god we are in the 21st century which has web browser addons  for translations :\

**Reply by o0LINNY0o** [reply]
unfortunately the french discord has also finished up with marlin and has moved to klipper.... in reality though klipper is the future so it only makes sense ... I got my stable linear advance on marlin so i dont see a need to update .... unless someone can shed some light on whats coming next ?

**Reply by RV-from-be** [reply]
not exactly, according to the dev.
![Capture d'écran 2022-10-13 15 16 52](https://user-images.githubusercontent.com/115715338/195607670-13e7abd4-bab3-439f-a7e1-c482cea10439.png)

**Comment by PLObiQ** [comment]
Hello
There is solution and it's called "[Professional Firmware for the Creality Ender 3 V2/S1 Printers](https://github.com/mriscoc/Ender3V2S1)"
I'm planning to flash this on my 3v2 today.

You welcome :)

**Reply by williamnauen** [reply]
Hi did your flash go well?

**Reply by PLObiQ** [reply]
Yes! I'm after couple of prints and i'm satisfied. There is one problematic thing for me, but despite of this it's good fw. Problem: timeout if you heat up bed and/or hotend and do nothing with printer for some time. I have stock glass bed and i'm waiting ~10 minutes after heat up to be sure that it's hot. Just because it's glass and it takes longer than heatup aluminum plate under the glass.
